src/ssl.c

```diff
@@ -5424,8 +5422,16 @@ int wolfSSL_CertManagerEnableOCSPStapling(WOLFSSL_CERT_MANAGER* cm)
 
     return ret;
 }
-
-
+#if defined(SESSION_CERTS)
+STACK_OF(WOLFSSL_X509)* wolfSSL_get_peer_cert_chain(const WOLFSSL* ssl)
```

Comment:
I see that it is tested against a NULL return, but have you verified it actually returns the expected peer chain?